### DISCUSSION TITLE
Right Click

### DISCUSSION THREAD

**Discussion by yurividal** [filebrowser/filebrowser#1031] [Ideas]
Any chance we can have right-click context menus? i know it would complicate a little bit the mobile experience, but for desktop, it would be a huge step.

**Comment by dantebarba** [comment]
+1

**Comment by jadsongmatos** [comment]
there is this project for nextcloud very cool, I think it would be interesting to have something like this.

![image](https://raw.githubusercontent.com/nextcloud/files_rightclick/master/screenshots/picture.png)

[https://apps.nextcloud.com/apps/files_rightclick](https://apps.nextcloud.com/apps/files_rightclick)

**Reply by damnshaneisthatu** [reply]
Long shot, but would love this functionality.
